### DISCUSSION TITLE
What place, if any, does benchmarking have in CI?

### DISCUSSION THREAD

**Discussion by hirehamir** [sharkdp/hyperfine#741] [General]
It probably doesn't make sense to run an hours-long benchmark on every push to a branch. But maybe it'd be useful to run small benchmarks to test against performance regressions.

**Comment by hirehamir** [comment]
Does anyone know of any GitHub Actions or workflows that do this sort of thing? In particular, I'm looking for ones that use `hyperfine`, just because it's an excellent CLI tool.

**Comment by sharkdp** [comment]
You might want to check out [Bencher](https://bencher.dev/) by @epompeii, which has a so-called adapter for `hyperfine`.

Full disclosure: Bencher is currently sponsoring hyperfine development (see README).
